src/ext/config.js

```diff
@@ -21,7 +21,7 @@ const exampleConfig = {
     "dev": true, // sets whether the configuration should only be used in development
     "sign": true, // sets whether tokens should be sent for signing
     "redeem": true, // sets whether tokens should be sent for redemption
-    "max-spends": 3, // for each host header, sets the max number of tokens that will be spent
+    "max-spends": 3, // for each host header, sets the max number of tokens that will be spent, 0 for unlimited
```

Comment:
Happy to have this ignored, but perhaps `max-spends` should be left undefined for infinite? Then we can just check `!SPEND_MAX` rather than `SPEND_MAX === 0`. My first thought when I saw `hcConfig["max-spends"] = 0` was that no spends were allowed.

Reply:
Great idea!